src/components/ViewRequest/Actions/ReworkRequest.tsx

```diff
@@ -24,7 +24,13 @@ const ReworkRequest = () => {
   const updateHandler = () => {
     if (request.data && currentStage && currentStage.previous !== "") {
       const newStage = currentStage.previous;
-      updateStage.mutate({ requestId, newStage });
+      updateStage.mutate({
+        requestId,
+        newStage,
+        ...(currentStage.previousEventTitle && {
+          eventTitle: currentStage.previousEventTitle,
+        }),
+      });
```

Comment:
Why not stick with the coding pattern you already have there?
```suggestion
      const eventTitle = currentStage.previousEventTitle;
      updateStage.mutate({
        requestId,
        newStage,
        eventTitle
      });
```